app/src/main/java/com/github/siela1915/bootcamp/NearbyHelpFragment.java

```diff
@@ -69,16 +112,41 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         return inflater.inflate(R.layout.fragment_nearby_help, container, false);
     }
 
+    @SuppressLint("MissingPermission")
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
```

Comment:
This `onViewCreated` method seems a bit lengthy which may affect the readability in the future refactoring can be considered, for example, create method like `addListeners()` to hold all the adding listener work. :)